ROADMAP.md

```diff
@@ -0,0 +1,32 @@
+# Kubeflow Distributed Training Operators 2020 Roadmap
+
+This document outlines the main directions on the Kubeflow Distributed Training Operators in 2020.
+
+## Maintenance and reliability
+
+We will continue developing capabilities for better reliability, scaling, and maintenance of production distributed training experiences provided by operators.
+
+* Enhance maintainability of operator common module. Related Issues: [#54](https://github.com/kubeflow/common/issues/54).
+* Migrate operators to use [kubeflow/common](https://github.com/kubeflow/common) apis.
+* Graduate MPI Operator, MXNet Operator and XGBoost Operator to v1.
+
+## Features
+
+To take advantages of other capabilities of job scheduler components, operators will expose more APIs for advanced scheduling. More features will be added to simplify usage like dynamic volume supports and git ops experiences. In order to make it easily used in the Kubeflow ecosystem, we can add more launcher KFP components for adoption.
+
+* Support dynamic volume provisioning for distributed training jobs. Realated Issue: [#19](https://github.com/kubeflow/common/issues/19).
+* MLOps - Allow user to submit jobs using Git repo without building container images.
+* Add Job priority and Queue in SchedulingPolicy for advanced scheduling in common operator. Realated Issue: [#46](https://github.com/kubeflow/common/issues/46).
+* Add pipeline launcher components for different training jobs. [pipeline#3445](https://github.com/kubeflow/pipelines/issues/3445).
+
```

Comment:
I am thinking that there might be many people submitting distributed training job directly using command line and job-operator. 
Maybe we can think about how to export the metrics and artifacts to the kubeflow/metadata in this case.
Or currently using metadata sdk in the training code is enough?

Reply:
@ChanYiLin 
User can check metrics or artifacts in separate UI, it's kind of isolated. The idea case is 

1. All job based metadata metrics or artifacts can be aggregated at the job level
2. Support rich metadata like bar chart, confusion metrics, roc curve, etc. Pipeline project has some, but I notice most of the training operator users may not use KFP, we can know from integration between KFP and operators.